packages/components/src/components/ViewSettings.tsx

```diff
@@ -57,6 +64,16 @@ export const ViewSettings: React.FC<{
         </div>
       </HeadedSection>
 
+      <div className="pt-8">
+        <HeadedSection title="Number Display Settings">
+          <InputItem
+            name="numberPrecision"
+            type="number"
+            register={register}
+            label="Chart Height (in pixels)"
```

Comment:
Label is incorrect. Also, I think it makes more sense to keep this under "General Display Settings" instead of having an extra HeadedSection here.

<img width="603" alt="Screen Shot 2022-10-12 at 12 16 07" src="https://user-images.githubusercontent.com/89368/195288660-18379f8e-691f-4d8b-bdb0-fb47d64231dd.png">